Coverage növelése

P6Z3F2 opened the pull request:
A store könyvtárral kapcsolatos függvények, melyek megtalálhatóak a doksiban teljesen le vannak fedve.
fix #12

P6Z3F2 committed:
Coverage növelése

A store könyvtárral kapcsolatos függvények, melyek megtalálhatóak a doksiban teljesen le vannak fedve.

pbalazs09 reviewed (changes requested):
Úgy lehetne a coverage-t legjobban növelni, ha a query-t is letesztelnéd. Azt ajánlom ne egyenként teszteld le a függvényeket, hanem legyen egy nagy teszt, viszont az minőségi.

P6Z3F2 committed:
Query tesztelése

Le lett tesztelve egy nagy tesztfüggvényben a querynek könyvtárnak a legfontosabb funkciói.

pbalazs09 reviewed (approved):
Látom most már jóval megnőtt a coverage, pontosan ilyenre gondoltam. Mehet a merge!
